src/test/kotlin/com/ing/dlt/zkkrypto/ecc/schnorr/SchnorrTest.kt

```diff
@@ -0,0 +1,180 @@
+package com.ing.dlt.zkkrypto.ecc.schnorr
+
+import com.ing.dlt.zkkrypto.ecc.EllipticCurvePoint
+import com.ing.dlt.zkkrypto.ecc.curves.AltBabyJubjub
+import com.ing.dlt.zkkrypto.ecc.curves.Jubjub
+import com.ing.dlt.zkkrypto.ecc.pedersenhash.PedersenHash
+import com.ing.dlt.zkkrypto.util.BitArray
+import org.junit.jupiter.api.Test
+import java.math.BigInteger
+
+
+class SchnorrSignatureTest {
+
+    @Test
+    fun `test raw message signing`(){
+
+        val schnorr = SchnorrSignature.zinc()
+
+        schnorr.generatePrivateKey()
+        schnorr.getPublicKeyFromPrivate()
+
+        println("Public Key:")
+        println("\tx coord = " + schnorr.publicKey.x.toString(16))
+        println("\ty coord = " + schnorr.publicKey.y.toString(16) + "\n")
+
+        val msg = 85.toByte()
+        val msgBytes = byteArrayOf(msg)
+
+        //val msg = "Foo bar pad to16"
+        //val msgBytes = msg.toByteArray()
+
+        val signature = schnorr.signRawMessage(msgBytes);
+
+        println("Signature:")
+        println("\tR:")
+        println("\t\tx coord = " + signature.r.x.toString(16))
+        println("\t\ty coord = " + signature.r.y.toString(16))
+        println("\ts:")
+        println("\t\tvalue = " + signature.s.toString(16))
+
+
+        val verified = schnorr.verifyRawMessage(msgBytes, signature)
+
+        if (verified)
+            println("\nSignature verified!")
+
+        assert(verified)
+    }
+
+    @Test
+    fun `test hashed message signing`(){
+
+        val schnorr = SchnorrSignature.zinc()
+
+        schnorr.generatePrivateKey()
+        schnorr.getPublicKeyFromPrivate()
+
+        println("Public Key:")
+        println("\tx coord = " + schnorr.publicKey.x.toString(16))
+        println("\ty coord = " + schnorr.publicKey.y.toString(16) + "\n")
+
+        val msg = 85.toByte()
+        val msgBytes = byteArrayOf(msg)
+
+        //val msg = "Foo bar pad to16"
+        //val msgBytes = msg.toByteArray()
+
+        val signature = schnorr.signHashedMessage(msgBytes);
+        val verified = schnorr.verifyHashedMessage(msgBytes, signature)
+
+        println("Signature:")
+        println("\tR:")
+        println("\t\tx coord = " + signature.r.x.toString(16))
+        println("\t\ty coord = " + signature.r.y.toString(16))
+        println("\ts:")
+        println("\t\tvalue = " + signature.s.toString(16))
+
+        if (verified)
+            println("Signature verified!")
+
+        assert(verified)
+    }
+
+    //@Test
```

Comment:
Either remove the comment or write `@Test @Ignore`

Reply:
Or better, upgrade to Junit5 as I did for ZKNotary, and tag these tests with 'benchmark' and create a 'benchmark' Gradle task, so that they can be run separately.